src/ChannelServer/World/Entities/Components/Buff.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using Melia.Channel.World.Entities;
+using Melia.Shared.Const;
+using Melia.Shared.Data.Database;
+
+namespace Melia.Channel.World
+{
+	/// <summary>
+	/// Modeled after Skill class
```

Comment:
This isn't a very useful description of what the class is and does, don't you think? If you hover over a `new Buff`, to see what that is, the text "this is like that other class" doesn't tell you much.